.chloggen/avillela-2734.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
We don't normally add changelog entries for documentation updates. Do you think it's worthwhile to have this one?

Reply:
+

Reply:
The CI build process erred out when I left it out 🤷‍♀️

Reply:
Removed the changelog.